nipype/interfaces/base/support.py

```diff
@@ -235,14 +238,166 @@ def version(self):
+
+    >>> from nipype.interfaces.afni import GCOR
+    >>> GCOR.help()  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
+    Wraps the executable command ``@compute_gcor``.
+    <BLANKLINE>
+    Computes the average correlation between every voxel
+    and ever other voxel, over any give mask.
+    <BLANKLINE>
+    <BLANKLINE>
+    For complete details, ...
 
     """
-    from ..fsl.model import load_template
-    iflogger.warning(
-        'Deprecated in 1.0.0, and will be removed in 1.1.0, '
-        'please use nipype.interfaces.fsl.model.load_template instead.')
-    return load_template(name)
+    from ...utils.misc import trim
+
+    docstring = []
+    cmd = getattr(cls, '_cmd', None)
+    if cmd:
+        docstring += ['Wraps the executable command ``%s``.' % cmd, '']
+
+    if cls.__doc__:
+        docstring += trim(cls.__doc__).split('\n') + ['']
+
+    allhelp = '\n'.join(
+        docstring +
+        _inputs_help(cls) + [''] +
+        _outputs_help(cls) + ['']  +
+        _refs_help(cls)
+    )
+    return allhelp.expandtabs(4)
+
+
+def _inputs_help(cls):
+    r"""
+    Prints description for input parameters
+
+    >>> from nipype.interfaces.afni import GCOR
+    >>> _inputs_help(GCOR)  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
+    ['Inputs::', '', '\t[Mandatory]', '\tin_file: (an existing file name)', ...
+
+    """
+    helpstr = ['Inputs::', '']
+    mandatory_keys = []
+    optional_items = []
+
+    if cls.input_spec:
+        inputs = cls.input_spec()
+        mandatory_items = list(inputs.traits(mandatory=True).items())
+        if mandatory_items:
+            helpstr += ['\t[Mandatory]']
+            for name, spec in sorted(mandatory_items):
+                helpstr += get_trait_desc(inputs, name, spec)
+
+        mandatory_keys = [item[0] for item in mandatory_items]
+        optional_items = ['\n'.join(get_trait_desc(inputs, name, val))
+                          for name, val in inputs.traits(transient=None).items()
+                          if name not in mandatory_keys]
+        if optional_items:
+            helpstr += ['\t[Optional]'] + optional_items
+
+    if not mandatory_keys and not optional_items:
+        helpstr += ['\tNone']
+    return helpstr
+
+
+def _outputs_help(cls):
+    r"""
+    Prints description for output parameters
+
+    >>> from nipype.interfaces.afni import GCOR
+    >>> _outputs_help(GCOR)  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
+    ['Outputs::', '', '\tout: (a float)\n\t\tglobal correlation value']
+
+    """
+    helpstr = ['Outputs::', '', '\tNone']
+    if cls.output_spec:
+        outputs = cls.output_spec()
+        outhelpstr = [
+            '\n'.join(get_trait_desc(outputs, name, spec))
+            for name, spec in sorted(outputs.traits(transient=None).items())]
+        if outhelpstr:
+            helpstr = helpstr[:-1] + outhelpstr
+    return helpstr
+
+
+def _refs_help(cls):
+    """Prints interface references."""
+    references = getattr(cls, '_references', None)
```

Comment:
```suggestion
    references = getattr(cls, 'references_', None)
```

Or we need to change all `references_` class attributes to `_references`.